spec/features/posts_appear_newest_first_spec.rb

```diff
@@ -1,10 +1,9 @@
 require 'rails_helper'
 
 RSpec.feature "Timeline", type: :feature do
-  let!(:post_0) { create(:post, message: "First post") }
-  let!(:post_1) { create(:post, message: "Second post") }
-
   let!(:user) { create(:user) }
+  let!(:post_0) { create(:post, message: "First post", user_id: user.id) }
```

Comment:
Move as much of this as possible to the user factory.

You don't need the user for the assertion, only for the setup, so find a way to use the factory to generate a post with a user. 

The end goal will be to have 
```ruby
 let!(:post_0) { create(:post) }
 let!(:post_1) { create(:post) }
``` 
where the factory will implement the rest of the setup.